### PR TITLE
Added docs on getting and setting AlchemicalNetwork weights, Task actioned weights, and Task priorities

### PR DESCRIPTION
**Pull request by dotsdl**
Also tossed in setting Task statuses to `deleted` and `invalid`.

### CODE REVIEW

File: docs/user_guide.rst [hunk 1]
```diff
@@ -210,6 +209,100 @@ In this way, actioning is an indicator of demand for a given :py:class:`~alchemi
...
+
+
+Setting the weight of actioned Tasks
+====================================
+
+As mentioned above, upon choosing an :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`, a compute service performs a weighted, random selection of :py:class:`~alchemiscale.storage.models.Task`\s actioned on that :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`.
+You can set the ``weight`` of an actioned :py:class:`~alchemiscale.storage.models.Task` to influence the likelihood that it will be picked up for compute relative to the other :py:class:`~alchemiscale.storage.models.Task`\s actioned on the given :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`.
+To set the ``weight`` of an actioned :py:class:`~alchemiscale.storage.models.Task` on an :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`, use :py:meth:`~alchemiscale.interface.client.AlchemiscaleClient.action_tasks` with the ``weight`` keyword argument::
+
+    >>> # get all networks that the givn Task is actioned on, with weights as dict values
```
Comment (ianmkenney) [inline docs/user_guide.rst:251]:
```suggestion
    >>> # get all networks that the given Task is actioned on, with weights as dict values
```

File: docs/user_guide.rst [hunk 2]
```diff
@@ -210,6 +209,100 @@ In this way, actioning is an indicator of demand for a given :py:class:`~alchemi
...
+These ``weight``\s can be set independently.
+Also, the :py:meth:`~alchemiscale.interface.client.AlchemiscaleClient.action_tasks` method is idempotent, so repeated calls will serve to set the ``weight`` to the value specified, even for already-actioned :py:class:`~alchemiscale.storage.models.Task`\s.
+
+
+Setting the priority of Tasks
+=============================
+
+The ``weight`` of an actioned :py:class:`~alchemiscale.storage.models.Task` influences how likely it is to be chosen among the other :py:class:`~alchemiscale.storage.models.Task`\s actioned on the given :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`.
+An orthogonal mechanism to ``weight`` is :py:class:`~alchemiscale.storage.models.Task` ``priority``, which is a property of the :py:class:`~alchemiscale.storage.models.Task` itself and introduces some determinism to when is executed relative to other :py:class:`~alchemiscale.storage.models.Task`\s actioned on the same :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`.
```
Comment (ianmkenney) [inline docs/user_guide.rst:270]:
```suggestion
Another compute selection mechanism, similar to ``weight``, is :py:class:`~alchemiscale.storage.models.Task` ``priority``, which is a property of the :py:class:`~alchemiscale.storage.models.Task` itself and introduces some determinism to when the :py:class:`~alchemiscale.storage.models.Task` is executed relative to other :py:class:`~alchemiscale.storage.models.Task`\s actioned on the same :external+gufe:py:class:`~gufe.network.AlchemicalNetwork`.
```

File: docs/user_guide.rst (hunk 1 above)
Comment (dotsdl) [reply docs/user_guide.rst:251]: Got it, thank you!

File: docs/user_guide.rst (hunk 2 above)
Comment (dotsdl) [reply docs/user_guide.rst:270]: Got it! Thank you!

**Pull request merged**